### ISSUE TITLE
DXE-3240 Terraform Akamai provider plugin crashes when trying to execute plan on app-sec (Web Security) 

### ISSUE THREAD

**Issue by Aravind1204** [akamai/terraform-provider-akamai#480]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
5.3.0

### Affected Resource(s)
Please list the resources as a list, for example:
- "akamai_appsec_advanced_settings_evasive_path_match"
If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.

### Terraform Configuration Files
```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### Debug Output
https://github.com/Aravind1204/terraform_error/blob/main/terraform_version_issue.txt

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.

### Expected Behavior
Expected behavior is that terraform returns the details about the specific security policy.

### Actual Behavior
The plugin.(*GRPCProvider).UpgradeResourceState request was cancelled

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform plan
![image](https://github.com/akamai/terraform-provider-akamai/assets/147028658/5ce32659-0827-4871-9787-7e4a7569e42e)

### Important Factoids
Are there anything atypical about your accounts that we should know? For example: Running in EC2 Classic? Custom version of OpenStack? Tight ACLs?

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
- GH-1234

**Comment by majakubiec** [comment]
Hi @Aravind1204,

Could you please share your config file? That would certainly help us reproducing the issue.

Please ensure to redact any sensitive information before sharing your configuration.

**Comment by Aravind1204** [comment]
Hi @majakubiec 

You can check the  pseudo code of our configuration here https://github.com/Aravind1204/terraform_error/blob/main/pseudo_code.txt

One more thing  I need to mention here is we configured / divided our security policies into 8 groups with same configuration, tested by removing 3 groups, that means with 5 groups and it worked.  Is this the issue with the Akamai terraform-provider cannot handle the more memory ?

**Comment by lkowalsk-akamai-com** [comment]
@Aravind1204 can you please test if this issue is also occring on 64bit system?

**Comment by chouraamine** [comment]
Hi @lkowalsk-akamai-com,
I have the same issue could you make it urgent case please ?
<img width="579" alt="image" src="https://github.com/akamai/terraform-provider-akamai/assets/115227803/13f786e6-e6b8-4a9c-accd-f73d1ec94555">

Thanks,

**Comment by lkowalsk-akamai-com** [comment]
Hi @chouraamine , fix for this issue is being tested and will be included in commnig release. There release should be available in couple of weekes.

**Comment by lkowalsk-akamai-com** [comment]
Hi, this was addressed in recent 5.6.0 release.
